lib/beacon/live_admin/live/page_editor_live/variants.ex

```diff
@@ -142,6 +160,16 @@ defmodule Beacon.LiveAdmin.PageEditorLive.Variants do
         </.button>
       </.modal>
 
+      <.modal :if={@show_delete_modal} id="confirm-delete" show>
```

Comment:
Can we add a friendly warning to let users know they need to publish the page in order to stop rendering the variant they just deleted?